brave/brave-core
# Depend on upstream WebTorrent

Fixes: https://github.com/brave/brave-browser/issues/5587
Fixes: https://github.com/brave/brave-browser/issues/856
Fixes: https://github.com/brave/brave-browser/issues/5489

This makes Brave depend on upstream WebTorrent instead of the fork. This also removes the need for forks of torrent-discovery and bittorrent-tracker.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

1. Visit https://webtorrent.io/free-torrents
2. Take turns clicking each torrent link, and also each magnet link.
3. When each of the torrent pages load, ensure the Start Download button starts the torrent download.
4. Ensure that the 'Copy Magnet Link' and 'Save .torrent File' links work.
5. Ensure that clicking on a video file or audio file causes a new tab to open and the media to play.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
